src/styles/utils/calendar-modification.scss

```diff
@@ -0,0 +1,27 @@
+@for $day from 1 through 31 {
+  .calendar__day:nth-child(#{$day})::before {
+    content: "#{$day}";
+  }
+}
+
+@each $day in mon, tue, wed, thu, fri, sat, sun {
+  .calendar--start-day-#{$day} {
+    .calendar__day:first-child {
+      margin-left: if($day == mon, 0,
+      if($day == tue, $cell-size + 1,
+      if($day == wed, ($cell-size * 2) + 2,
+      if($day == thu, ($cell-size * 3) + 3,
+      if($day == fri, ($cell-size * 4) + 4,
+      if($day == sat, ($cell-size * 5) + 5,
+      $cell-size * 6 + 6))))));
+    }
```

Comment:
It is overcomplicated. Please pay attention to the previous comments and task description
![image](https://github.com/mate-academy/layout_calendar/assets/96409563/781d36a2-ef82-4992-87c5-481472546e6e)
https://sass-lang.com/documentation/at-rules/control/each/#with-maps